src/ControlSystem/Event.hpp

```diff
@@ -78,6 +78,8 @@ class Event : public ::Event {
   static constexpr bool factory_creatable = false;
   Event() = default;
 
+  using compute_tags_for_observation_box = tmpl::list<>;
```

Comment:
Could you add a bit of documentation either here, or to the class docs, so a person implementing a subclass knows they are supposed to add this alias, and they have an idea what it is